tests/Entity/MessageManagerTest.php

```diff
@@ -151,6 +151,22 @@ public function testGetPagerWithInProgressMessages(): void
             ->getPager(['state' => MessageInterface::STATE_IN_PROGRESS], 1);
     }
 
+    public function testFindByTypes(): void
+    {
+        $self = $this;
```

Comment:
IIRC correctly, this is only needed for older PHPUnit versions, but we are running our test suite with PHPUnit 7.

Can you please double check this?

Reply:
i will ;) and will add comment here

Reply:
true. no need for this anymore. Do you want i update the whole class with this change?

Reply:
Would be great, but please do it at least in n extra commit, extra PR would bring bonus points 😄

Reply:
will do an extra PR ;)

Reply:
@OskarStark i have prepared an extra MR to fix this point. Does it have to target master too or 3.x is better?